src/api/app/views/webui2/webui/project/_breadcrumb_items.html.haml

```diff
@@ -1,27 +1,31 @@
 = render partial: 'webui/main/breadcrumb_items'
-- if current_page?(projects_path)
+- if current_page?(projects_path) || current_page?(project_list_path) || current_page?(project_list_public_path)
   %li.breadcrumb-item.active{ 'aria-current' => 'page' }
     Projects
 - else
   %li.breadcrumb-item
     = link_to 'Projects', projects_path
-  %li.breadcrumb-item
-    = link_to @project, project_show_path(@project)
-  - if current_page?(project_show_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Overview
-  - elsif current_page?(project_monitor_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Monitor
-  - elsif current_page?(project_requests_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Requests
-  - elsif current_page?(project_users_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Users
-  - elsif current_page?(project_config_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Configuration
-  - elsif current_page?(project_status_path(@project))
+  - if current_page?(project_list_all_path)
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Status
+      List of all projects
```

Comment:
`List of All Projects` was right in the first place. I use https://capitalizemytitle.com/ when I'm not sure about capitalization.

Reply:
Done.